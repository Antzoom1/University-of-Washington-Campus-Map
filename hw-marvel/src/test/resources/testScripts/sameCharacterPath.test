# Tests if a path has the same startPoint, endPoint, then it won't print anything

LoadGraph staffSuperheroes staffSuperheroes.csv

FindPath staffSuperheroes Grossman-the-Youngest-of-them-all Grossman-the-Youngest-of-them-all