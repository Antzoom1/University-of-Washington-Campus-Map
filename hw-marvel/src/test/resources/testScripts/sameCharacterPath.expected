# Tests if a path has the same startPoint, endPoint, then it won't print anything

loaded graph staffSuperheroes

path from Grossman-the-Youngest-of-them-all to Grossman-the-Youngest-of-them-all: